### DISCUSSION TITLE
Response should be compressed with Zopfli when gzip compression is requested.

### DISCUSSION THREAD

**Discussion by michael-voit** [plausible/analytics#1431] [Anything Else]
After starting using your tool in a website, we get the following hint from Webhint:

> hint #1: Response should be compressed with Zopfli when gzip compression is requested.
> https://plausible.io/js/plausible.js 

Is that possible with your infrastructure, or how can we deal with it?

**Comment by ukutaht** [comment]
Thanks, I hadn't heard of Zopfli.

Currently we rely on Netlify to do standard gzip compression. It wouldn't hurt to use Zopfli to get the 3-8% improvement mentioned here: https://webhint.io/docs/user-guide/hints/hint-http-compression/

3-8% sounds like a marginal improvement but I will take a look soon since we're planning to migrate CDN tools.
